docs/topics/compiler-reference.md

```diff
@@ -156,6 +156,12 @@ environment.
 
 Use a custom JDK home directory to include into the classpath if it differs from the default `JAVA_HOME`.
 
+### -jdk-release
+
+Specify the target version of the generated JVM bytecode. Limit the API of the JDK in the classpath to the specified Java version. 
+Automatically sets [`-jvm-target version`](#jvm-target-version). Works only for [OpenJDK distributions](https://openjdk.java.net/projects/jdk/17/).
+Possible values are `1.8`, `9`, `10`, `11`, `12`, `13`, `14`, `15`, `16`, and `17`. The default value is `%defaultJvmTargetVersion%`.
```

Comment:
```suggestion
Possible values are `1.8`, `9`, `10`, ..., `18`. The default value is `%defaultJvmTargetVersion%`.
```

Reply:
@koshachy do we want to omit versions this way?

Reply:
This is inspired by https://github.com/JetBrains/kotlin/commit/7952e8b28ffca748b8e140042c192e0fc7e1d655#diff-63f8fc6179222fd81cf59c707e56cc01578d60cde1e88e6f3526e804204785e1R84. There's a new JDK version every 6 months, so I decided we don't want to add more values to an already absurdly long line

Reply:
Ok, I fixed everywhere.